tlog : validate data+parity against given servers.

iwanbk opened the pull request:
Fixes #507

GlenDC commented on tlog/stor/config_source.go:
```diff
@@ -21,6 +23,12 @@ func ConfigFromConfigSource(source config.Source, vdiskID, privKey string, dataS
 	}
 
+	minServerNum := dataShards + parityShards
+	if minServerNum > len(zsc.Servers) {
```
small nit pick, but I would store `len(zsc.Servers)` in a local variable here, as you're going to reuse it. Not that getting the length is expensive, but still...

iwanbk commented:
> (thus with 4/2 config)
> 
> I still don't get a complain, even though it should clearly exit with error on startup (of a vdisk), as the amount of servers I specify does not match the 4/2 data/parity config?

Fixed, forgot to `return`

> small nit pick, but I would store len(zsc.Servers) in a local variable here, as you're going to reuse it. Not that getting the length is expensive, but still...

fixed

GlenDC commented:
> Fixed, forgot to return

I can confirm it does indeed now return an error when using the wrong config.

My config however was correct as expected, so that wasn't my issue. However I don't have the issue anymore of yesterday, it somehow just went away so for now can't say much about what the issue might have been, as I have no clue.

---

Either way, your PR fixes the issue, so merged.
